stable/yugaware/templates/configs.yaml

```diff
@@ -176,6 +176,12 @@ data:
       location / {
         proxy_pass http://{{ eq .Values.ip_version_support "v6_only" | ternary "[::1]" "127.0.0.1" }}:9000;
       }
+{{- if .Values.nginx.db_node_proxy_enabled }}
+      location ~ "^/universes/.+/proxy/(.+):(7000|9000|9300|12000|13000)/(metrics|prometheus-metrics)$" {
```

Comment:
I think we should validate that this is an IP and block IPs in the 169.254 range as these are used by cloud metadata server and can contain auth credentials.